src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/mapper/EnvelopeMapper.java

```diff
@@ -126,6 +126,7 @@ private static DocumentType mapDocumentType(InputDocumentType inputDocumentType)
             case COVERSHEET:
                 return DocumentType.COVERSHEET;
             case FORM:
+            case SSCS1:
```

Comment:
I think this means we should remove from schema validation if present and require for it to be strictly just `form`

Reply:
do you mean metafile-schema.json? @doncem

Reply:
yes this is the only schema we have. but this change is very serious as it purely depends on a contract among each service. plus we insist to be like that anyway

Reply:
We have to keep accepting SSCS1 for now. We'll remove it once Exela has done their change. Unfortunately, for the time being we have to deal with SSCS1 document types coming in metadata files.

Reply:
That's what I thought, Thanks for clarifying this @lgonczar